site/docs/v1/tech/identity-providers/samlv2-idp-initiated/okta.adoc

```diff
@@ -0,0 +1,131 @@
+---
```

Comment:
I read through this page doing a once-over and looks good to me. I'd like to take the time to actually execute this example at some point, but I'm trusting the author has done this to confirm :)

Reply:
A few times, a few times. :)